crates/ruff_linter/src/rules/flake8_todos/snapshots/ruff_linter__rules__flake8_todos__tests__missing-todo-link_TD003.py.snap

```diff
@@ -10,40 +10,49 @@ TD003.py:9:3: TD003 Missing issue link on the line following this TODO
 10 | # link after it
    |
 
-TD003.py:12:3: TD003 Missing issue link on the line following this TODO
+TD003.py:18:3: TD003 Missing issue link on the line following this TODO
```

Comment:
The formatting of this chunk of the snapshot seems incorrect. Looks like it's combining two checks into a single section. Needs fixing.